Compilation error:  'phi_grid' not specified in enclosing 'parallel'

I get the following error when compiling the HEAD version with OpenMP support:

```
f77 -m64 -cpp -O3 -funroll-loops -ffinite-math-only -fno-trapping-math -ffree-line-length-none -fbacktrace -fopenmp -c -DPOINTER_SIZE=8 mol_transfer.f90 -o mol_transfer.o
mol_transfer.f90:320:0:

  320 |                     x0 = r_grid(icell) * cos(phi_grid(icell))
      | 
Error: 'phi_grid' not specified in enclosing 'parallel'
mol_transfer.f90:300:0:

  300 |         !$omp shared(nTrans_tot,tab_Trans)
      | 
Error: enclosing 'parallel'
make: *** [Makefile:235: mol_transfer.o] Error 1
```

--- a/src/mol_transfer.f90
+++ b/src/mol_transfer.f90
@@ -295,7 +295,7 @@ subroutine NLTE_mol_line_transfer(imol)
         !$omp private(id,iray,rand,rand2,rand3,x0,y0,z0,u0,v0,w0,w02,srw02) &
         !$omp private(argmt,n_iter_loc,lconverged_loc,diff,norme,iv,icell,factor) &
         !$omp shared(imol,stream,n_rad,nz,n_az,n_rayons,iray_start,Doppler_P_x_freq,tab_nLevel,n_level_comp) &
-        !$omp shared(deltaVmax,ispeed,r_grid,z_grid,lcompute_molRT,lkeplerian,n_cells) &
+        !$omp shared(deltaVmax,ispeed,r_grid,z_grid,phi_grid,lcompute_molRT,lkeplerian,n_cells) &
         !$omp shared(tab_speed,lfixed_Rays,lnotfixed_Rays,pop_old,pop,labs,n_speed,max_n_iter_loc,etape,pos_em_cellule) &
         !$omp shared(nTrans_tot,tab_Trans)
         !$omp do schedule(static,1)

